src/DAMA/DoctrineTestBundle/PHPUnit/PHPUnitListener.php

```diff
@@ -20,6 +21,7 @@ public function endTest(\PHPUnit\Framework\Test $test, float $time): void
 
     public function startTestSuite(\PHPUnit\Framework\TestSuite $suite): void
     {
-        StaticDriver::setKeepStaticConnections(true);
+        $keepStatic = !is_subclass_of($suite->getName(), DoNotRunInTransactionInterface::class);
```

Comment:
so this will not work properly because this is done once BEFORE all tests are running.

It's a `TestSuite` and not a `TestCase` :wink: 

Since you want to do this per test case this will not work.